README.md

```diff
@@ -1 +1,52 @@
-# doppler-webapp
\ No newline at end of file
+# Doppler new webapp with reactjs
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Install
+
+In the project directory, you can run:
+
+`npm install` or `yarn install`
```

Comment:
Why not define one of them right now? I think that it is not recommendable mixing both.

Reply:
BTW (I prefer Yarn)

Reply:
yes i use Yarn but i don't know if other prefeer use npm, but we can choose one and use only this